BoardConfig.mk

```diff
@@ -59,6 +59,16 @@ BOARD_BOOTIMAGE_PARTITION_SIZE := 28483584
 BOARD_BUILD_SYSTEM_ROOT_IMAGE := false
 BOARD_USES_RECOVERY_AS_BOOT := true
 
+
+# Verified boot
+ifeq ($(ENABLE_AVB),true)
+BOARD_AVB_ENABLE := true
+BOARD_AVB_ALGORITHM := SHA512_RSA4096
+BOARD_AVB_SYSTEM_ADD_HASHTREE_FOOTER_ARGS := --do_not_generate_fec
+BOARD_AVB_VENDOR_ADD_HASHTREE_FOOTER_ARGS := --do_not_generate_fec
+endif
+
+
```

Comment:
Seems like not needed empty lines before and after this block. If it is made intentionally - OK.

Reply:
Done